src/React/Redux.purs

```diff
@@ -39,7 +39,7 @@ import React as React
 
 import Unsafe.Coerce (unsafeCoerce)
 
-type ConnectClass' state props = ConnectClass state () props
+type ConnectClass' state props = ConnectClass state (Record ()) props
```

Comment:
why not just `{}` instead of `Record ()`?

Reply:
Good call. I will update this.